### PR TITLE
Fix codesniffer complains on some apps

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix some apps from #1349 

To test run:

```
cd build/codesniffer
./vendor/bin/phpcs -s -p --extensions=php --standard=nextcloud-unused.xml ../../apps/<appid>/
```

### CODE REVIEW

**Comment by nickvergessen** [comment]
cc @rullzer

File: apps/twofactor_backupcodes/tests/Integration/Service/BackupCodeStorageTest.php [hunk 1]
```diff
@@ -40,7 +39,7 @@ class BackupCodeStorageTest extends TestCase {
 	protected function setUp() {
 		parent::setUp();
 
-		$this->storage = OC::$server->query(BackupCodeStorage::class);
+		$this->storage = \OC::$server->query(BackupCodeStorage::class);
```
Comment (ChristophWurst) [inline apps/twofactor_backupcodes/tests/Integration/Service/BackupCodeStorageTest.php]: why can't we import the `OC` class with `use`? Is there anything special about that one?

Comment (nickvergessen) [inline apps/twofactor_backupcodes/tests/Integration/Service/BackupCodeStorageTest.php]: Not sure, but I think use-ing a non-namespace class is awkward anyway

**Comment by MorrisJobke** [comment]
Rebased to fix failing test

**Comment by rullzer** [comment]
I like it!
LGTM

**Comment by ChristophWurst** [comment]
:+1:

**Pull request merged**